src/controllers/v1/config.js

```diff
@@ -0,0 +1,20 @@
+const httpStatusCode = require('@generics/http-status')
+const common = require('@constants/common')
+const configsService = require('@services/config')
+const responses = require('@helpers/responses')
+
+module.exports = class configs {
+	async list(req) {
+		try {
+			const configList = await configsService.list(req.decodedToken.organization_id)
+
+			return responses.successResponse({
```

Comment:
@adithyadinesh0412 just return whatever coming from the service, status code and message define in the service itself